tests/keras/backend/backend_test.py

```diff
@@ -1702,6 +1702,56 @@ def test_sparse_concat(self):
             assert k_s_d.shape == k_d.shape
             assert_allclose(k_s_d, k_d, atol=1e-05)
 
+    @pytest.mark.skipif((K.backend() != 'mxnet'),
+                        reason='Testing only for MXNet backend')
+    def test_sparse_sum(self):
+        x_d = np.array([0, 7, 2, 3], dtype=np.float32)
+        x_r = np.array([0, 2, 2, 3], dtype=np.int64)
+        x_c = np.array([4, 3, 2, 3], dtype=np.int64)
+
+        x_sparse = sparse.csr_matrix((x_d, (x_r, x_c)), shape=(4, 5))
+        x_dense = x_sparse.toarray()
+
+        W = np.random.random((5, 4))
+
+        k_s = K.eval(K.sum(K.variable(x_sparse), axis=0))
```

Comment:
add check for `is_sparse` like in the test sparse concat
```
k_s = k.concatenate([k.variable(x_sparse_1), k.variable(x_sparse_2)])
assert k.is_sparse(k_s)
```

Reply:
For mxnet backend, currently sparse tensor is not created in K.variable, so this is comparing the same results. Adding `assert k.is_sparse()` will ensure sparse tensor is used.

Reply:
added implementation of sparse tensor as well now